src/Board.svelte

```diff
@@ -243,6 +245,7 @@
       transition:fade={{ duration: 200 }}
       class="d-none d-lg-block scroll h-100"
     >
+      <IceBreaker data-name="ice-breaker-message" class="w-50" />
```

Comment:
Here the 'data-name' property is passed as a kind of argument to the svelte component.
In your component you need to declare the variable and then use it on an element within the component.
I'd recommend doing it differently however, let me make another comment in the right place.